packages/babel-code-frame/test/index.js

```diff
@@ -17,7 +17,7 @@ suite("babel-code-frame", function () {
     ].join("\n"));
   });
 
-  test("optional column number", function () {
+  it("optional column number", function () {
```

Comment:
I always thought that the function is called `it` to help writing sentences that `describe` the behavior. Like `it('allows column numbers to be optional')`. I kinda always cringe when people use `it` the `test` way, but whatever :)